include/boost/geometry/algorithms/detail/buffer/buffered_piece_collection.hpp

```diff
@@ -808,14 +815,14 @@ struct buffered_piece_collection
         }
     }
 
-    static inline void determine_properties(piece& pc)
+    inline void determine_properties(piece& pc) const
```

Comment:
since you added `const` here you could also added to other non-static function members e.g. the other specialization of `determine_properties`

Reply:
Thanks, I missed your comments. I can have a look there indeed, better if they are mutually consistent.
And I will ook at the long line.

Reply:
Sorry, the other one cannot be made `const` because it modifies `m_pieces`.
And this one should not be `static` anymore because it accesses member strategies.
So current code is OK